packages/ui-components/src/components/range/range.base.scss

```diff
@@ -37,25 +39,25 @@
 	outline: none;
 	margin: 0;
 	border-radius: $spacing;
-	background: var(--slider-background-empty);
 	user-select: none;
+	background: var(--slider-background-empty);
 
 	&::-webkit-slider-thumb {
-		border: calc($spacing - 1px) solid kv-color('neutral-0');
-		height: calc(var(--button-selector-height) + 5px);
-		width: calc(var(--button-selector-width) + 5px);
-		border-radius: 50%;
+		border: calc($spacing - 1px) solid var(--thumb-border-color);
 		background: var(--slider-background-filled);
+		height: calc(var(--range-selector-radius) + 5px);
+		width: calc(var(--range-selector-radius) + 5px);
+		border-radius: 50%;
 		cursor: pointer;
 		appearance: none;
 	}
 
 	&::-moz-range-thumb {
-		border: $spacing solid kv-color('neutral-0');
-		height: var(--button-selector-height);
-		width: var(--button-selector-width);
-		border-radius: 50%;
+		border: $spacing solid var(--thumb-border-color);
 		background: var(--slider-background-filled);
+		height: var(--range-selector-radius);
```

Comment:
I'm wondering if there's any css prop that could help us in these cases where width and height are the same...

Not necessarily a need but a nice to have if we could, since we're defining size using a radius, I've found that using just the height or width and then using the `aspect-ratio` css property with value `1` would work, but it would still add 2 lines instead of minimising it to just one, so I don't think it's worth it.